detect_analysis.R

```diff
@@ -0,0 +1,47 @@
+---
+title: "Reporting systems sub-analysis"
+author: "Emily Linebarger"
+date: "2/17/2021"
+output: pdf_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+library(data.table)
+
+dt = fread("C:/Users/eklin/Documents/repos/data_557/prepped_data/sixmonth.csv")

```

Comment:
To make this easier for others to reproduce, can this be made into a relative path?

Reply:
Yep! Good call.